app/models/download_acceptance.rb

```diff
@@ -0,0 +1,28 @@
+class DownloadAcceptance
```

Comment:
Put a comment outlining general use for the class

Reply:
2e76f7c